Yoast/YoastSEO.js
# Add whitespace structured element

## Summary

This PR can be summarized in the following changelog entry:

* [not-user-facing] Adds a Whitespace class for use within the structured tree.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test`, check if all the tests run and the Whitespace class has a coverage of 100%.

Fixes #2027
